### DISCUSSION TITLE
Does KaTeX support individual labels for lines in aligned?

### DISCUSSION THREAD

**Discussion by pgeipi10** [KaTeX/KaTeX#3183] [General]
Is it possible to give each equation in an "align" an individual equation number? 
<img width="1246" alt="Screen Shot 2021-08-22 at 7 15 01 PM" src="https://user-images.githubusercontent.com/7796257/130364141-96d9abb4-2463-4b0f-afb2-2a3e66b51515.png">

**Comment by edemaine** [comment]
Yes, as of [v0.13.0](https://github.com/KaTeX/KaTeX/blob/master/CHANGELOG.md#v0130). [Here's a demo](https://katex.org/?data=%7B%22displayMode%22%3Atrue%2C%22leqno%22%3Afalse%2C%22fleqn%22%3Afalse%2C%22throwOnError%22%3Atrue%2C%22errorColor%22%3A%22%23cc0000%22%2C%22strict%22%3A%22warn%22%2C%22output%22%3A%22htmlAndMathml%22%2C%22trust%22%3Afalse%2C%22macros%22%3A%7B%22%5C%5Cf%22%3A%22%231f(%232)%22%7D%2C%22code%22%3A%22%5C%5Cbegin%7Balign%7D%5Cna%20%26%3Db%2Bc%2Bd%2Be%20%5C%5C%5C%5C%5Cna%2Bb%2Bc%2Bd%20%26%3D%20e%5Cn%5C%5Cend%7Balign%7D%22%7D) for the following input:

```latex
\begin{align}
a &=b+c+d+e \\
a+b+c+d &= e
\end{align}
```

![image](https://user-images.githubusercontent.com/2218736/130364250-9e52ba6f-f0cc-4acb-b28f-b6f6514717c5.png)

**Reply by pgeipi10** [reply]
Thank you, Erik!

To what degree do I get to control what the labels say? For example, can I have the labels say (13.7) and (13.24)?

**Reply by edemaine** [reply]
That sounds like the `\tag` feature, which is currently blocked on #2952, which is awaiting code review. Unfortunately, it's difficult to predict exactly when that will be released.
